ydb/mvp/meta/meta_cloud.h

```diff
@@ -51,7 +51,7 @@ class THandlerActorMetaCloudGET : THandlerActorYdbc, public NActors::TActorBoots
         NActors::TActorId actorId = ctx.SelfID;
 
         {
-            Location.GetTableClient(Request, NYdb::NTable::TClientSettings().Database(Location.RootDomain))
+            Location.GetTableClient(Request, NYdb::NTable::TClientSettings().Database(Location.RootDomain), TMVP::MetaDatabaseTokenName)
```

Comment:
is there a way to put a token into client settings?

Reply:
it's complecated because it's part of public api